extension/static/manifest.json

```diff
@@ -0,0 +1,83 @@
+{
+  "manifest_version": 2,
+  "name": "waves_wallet",
+  "version": "0.0.1",
+  "description": "A Liquid Wallet",
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "waves-wallet@comit.network"
+    }
+  },
+
+  "background": {
+    "page": "background.html"
+  },
+
+  "icons": {
+    "48": "icons/favicon.svg",
+    "96": "icons/favicon.svg"
+  },
+
+  "browser_action": {
+    "default_title": "waves_wallet",
+    "browser_style": true,
+    "default_popup": "popup.html"
+  },
+  "permissions": [
+    "<all_urls>",
+    "activeTab",
+    "alarms",
+    "bookmarks",
+    "browserSettings",
+    "browsingData",
+    "captivePortal",
+    "clipboardRead",
+    "clipboardWrite",
+    "contextMenus",
+    "contextualIdentities",
+    "cookies",
+    "dns",
+    "downloads",
+    "downloads.open",
+    "find",
+    "geolocation",
+    "history",
+    "identity",
+    "idle",
+    "management",
+    "menus",
+    "menus.overrideContext",
+    "nativeMessaging",
+    "notifications",
+    "pkcs11",
+    "privacy",
+    "proxy",
+    "search",
+    "sessions",
+    "storage",
+    "tabHide",
+    "tabs",
+    "theme",
+    "topSites",
+    "unlimitedStorage",
+    "webNavigation",
+    "webRequest",
+    "webRequestBlocking"
+  ],
+  "content_scripts": [
+    {
+      "matches": [
+        "*://*/*"
```

Comment:
:question: Should we only insert the content script on more specific URLs?

Reply:
Hard to say which one. We should be using dynamic injection and inject only when confirmed through the webapplication.